resources/_partial/_common.rb

```diff
@@ -0,0 +1,8 @@
+property :user,
+        String,
+        description: ''
+
+property :root_path,
+        String,
+        default: lazy { Chef::Rbenv::Helpers.root_path(node, user) },
+        description: ''
```

Comment:
~~If you're using partials then you need to bump up the `chef_version` to 16 I believe.~~
I see below that you've commented that out in the resources for what I assume a future release?

Reply:
yup, doing a major release with Chef 15.3 support, then doing a Chef 16 release for partials 👍🏼

Reply:
Not sure if it's worth adding a comment in those resources where you commented it out or not so we know that. But maybe you should do that.